src/groups/repository/postgres.repository.ts

```diff
@@ -0,0 +1,46 @@
+import { InjectModel } from '@nestjs/sequelize';
+import { Group } from '../model/groups.model';
+import { CreateGroupDto } from '../dto/create-group.dto';
+import { UpdateGroupDto } from '../dto/update-group.dto';
+import { GroupsRepository } from './groups.repository';
+
+export class PostgresGroupsRepository implements GroupsRepository {
+  constructor(@InjectModel(Group) private GroupModel: typeof Group) {}
+  async create(dto: CreateGroupDto): Promise<Group> {
+    const group = await this.GroupModel.create({ ...dto });
+    return group;
+  }
+
+  async findByID(id: string): Promise<Group> {
+    try {
+      const group = await this.GroupModel.findByPk(id);
+      return group;
```

Comment:
why saving result to the constant if you not using it for anything? better to return it right away
```suggestion
     return await this.GroupModel.findByPk(id);
```